website/blog/2024-06-11-run-mongodb-workloads-aiven-postgresql.md

```diff
@@ -0,0 +1,278 @@
+---
+slug: run-mongodb-workloads-aiven-postgresql
+title: 'Run MongoDB Workloads on Aiven for PostgreSQL'
+authors: [alex]
+description: >
+  In this blog, we'll show you how to run MongoDB workloads on Aiven for PostgreSQL using FerretDB.
+image: /img/blog/ferretdb-aiven.jpg
+tags: [tutorial, postgresql tools, open source, cloud]
+---
+
+![Run MongoDB Workloads on Aiven for PostgreSQL](/img/blog/ferretdb-aiven.jpg)
+
+Adopting open-source solutions is a strategic move for modern businesses.
+For those looking to migrate from MongoDB, [FerretDB](https://www.ferretdb.com/) is a truly open source alternative that lets you run MongoDB workloads on [PostgreSQL](https://www.postgresql.org/).
+
+<!--truncate-->
+
+With [Aiven for PostgreSQL](https://aiven.io/postgresql), you can set up a reliable backend for FerretDB to run your MongoDB workloads.
+Aiven provides a unified, cloud-agnostic platform that lets you tap into several critical features, including PostgreSQL extensions, high availability and performance, as well as integration with your existing data infrastructure.
+
+In this blog, we'll show you how to run MongoDB workloads on Aiven for PostgreSQL using FerretDB.
+
+## Prerequisites
+
+Before you start, ensure you have the following set up:
+
+- [Aiven account](https://aiven.io/)
+- `psql`
+- [Docker](https://www.docker.com/)
+- `mongosh`
+
+## Set up Aiven for PostgreSQL
+
+First, create an Aiven account if you don't have one.
+From the Aiven dashboard, [create a PostgreSQL service](https://aiven.io/docs/products/postgresql).
+
+![Aiven for PostgreSQL service](/img/blog/aiven-postgres.png)
+
+Once it's set up, your Aiven for PostgreSQL connection string should look like this:
+
+```text
+postgres://<username>:<password>@<host>:port/database?sslmode=require
```

Comment:
I guess it's either
```suggestion
postgres://<username>:<password>@<host>:<port>/<database>?sslmode=require
```

or

```suggestion
postgres://username:password@host:port/database?sslmode=require
```

So it looks consistent :) Let's probably use the same format we had in the previous blog posts or in the docs?